LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [2016] [sksamuel]
+   Copyright [2025] [sksamuel]
```

Comment:
I think this part shouldn't even be modified on the license.

Thanks for the bumpb anyways :D